scripts/check-versions.py

```diff
@@ -0,0 +1,269 @@
+#!/usr/bin/env python3
+__author__ = "Johannes Schrimpf"
+__copyright__ = "Copyright 2017, Blueye Robotics AS"
+__credits__ = ["Johannes Schrimpf"]
+__license__ = "GPLv3"
+
+"""
+check-version.py is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+check-version.py is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+See <https://www.gnu.org/licenses/gpl-3.0.html> for full license text.
+"""
+
+import os
+import urllib.request
+import sys
+import re
+import hashlib
+from distutils.version import LooseVersion
+import yaml
+
+BASE_DIR = "../recipes-ros"
+DEBUG = True
+EXCLUDE = ["packagegroups"]
+DISTRO = "indigo"
+DIST_FILE = "https://raw.githubusercontent.com/ros/rosdistro/master/" + DISTRO+ "/distribution.yaml"
+dist_raw = None
+
+
+class MoveRepoExcetion(Exception):
+    pass
+
+def print_debug(text):
```

Comment:
Quite often even trivial scripts tend to evolve to something bigger. It's beneficial in long run to use the `logging` and `argparse` modules instead of home grown solutions from the beginning.